crates/sdk/src/migrations.rs

```diff
@@ -431,6 +506,46 @@ impl Display for UpdateStatus {
     }
 }
 
+/// Check if a scheduled migration should take place at this block height.
+/// If so, apply it to the DB.
+pub fn commit<D: DB>(
+    db: &D,
+    height: BlockHeight,
+    mut migration: Option<ScheduledMigration<D::Migrator>>,
+) -> Option<ScheduledMigration<D::Migrator>>
+where
+    D::Migrator: DeserializeOwned,
+{
+    let maybe_migration = migration.take();
```

Comment:
don't think you need to do this here as far as I can tell
```suggestion
    migration: Option<ScheduledMigration<D::Migrator>>,
) -> Option<ScheduledMigration<D::Migrator>>
where
    D::Migrator: DeserializeOwned,
{
    let maybe_migration = migration;
```